crypto/evp/keymgmt_lib.c

```diff
@@ -296,18 +296,22 @@ void evp_keymgmt_util_cache_keyinfo(EVP_PKEY *pk)
     if (pk->keydata != NULL) {
         int bits = 0;
         int security_bits = 0;
+        int security_category = -1;
```

Comment:
Maybe be consistent with the other ones and use 0 here..

Reply:
`-1` represents an algorithm with an unknown security category.  Using `0` here is wrong.